build.gradle

```
@@ -65,7 +65,7 @@ publishing {
 
 dependencies {
     implementation "org.eclipse.lsp4j:org.eclipse.lsp4j:0.9.0"
-    implementation "software.amazon.smithy:smithy-model:1.1.0"
+    implementation "software.amazon.smithy:smithy-model:1.+"
```

Could this use the Maven-style ranges instead?  That'd help ensure compatibility.
```suggestion
    implementation "software.amazon.smithy:smithy-model:[1.0, 2.0["
```

---

Adressed in 2728292